packages/govern-console/src/containers/CreateDAO/CreateDaoForm.tsx

```diff
@@ -122,25 +123,33 @@ const CreateDaoForm: React.FC<FormProps> = ({
   setCreatedDaoRoute,
   cancelForm,
 }) => {
+  const { enqueueSnackbar } = useSnackbar();
   const context: any = useWallet();
-  const { chainId, status, provider } = context;
+  const { chainId, provider, isConnected } = context;
 
   const { control, handleSubmit, watch, setValue } = useForm<FormInputs>();
 
   const connectedChainId = useMemo(() => {
-    if (chainId === ChainId.RINKEBY && status === 'connected')
+    if (chainId === ChainId.RINKEBY && isConnected)
       return ChainId.RINKEBY;
 
     return ChainId.MAINNET;
-  }, [chainId, status]);
+  }, [chainId, isConnected]);
 
   // use appropriate default config
   useEffect(() => {
     const _config: DaoConfig = DEFAULT_DAO_CONFIG[connectedChainId];
+    console.log('defultconfig:', _config)
     setValue('daoConfig', _config);
   }, [connectedChainId]);
 
   const submitCreateDao = async (params: FormInputs) => {
+    if (!isConnected) {
```

Comment:
do we need this check ? having a button disabled if it's not connected is much better and enough.